packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -284,6 +285,16 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
       suggestedDisplayValue: itemValue,
       suggestionsVisible: this.input.value !== '' && this.input.inputElement === document.activeElement
     });
+
+    /**
+     * If user exits the input box before suggestions are returned,
+     * select the first result upon promise resolution, if a suggestion
+     * is available.
+     */
+    if (this.suggestionStore.hasSelectedSuggestion() &&
```

Comment:
I think it should be an option then, maybe. Part of me feels that the picker should work more like a search box.